.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: ruby
+rvm:
+  - 2.4.1
+before_script:
+  - bundle update --bundler
+  - chmod +x ./script/cibuild
+script: ./script/cibuild
+branches:
+  only:
+  - gh-pages     # test the gh-pages branch
+  - /pages-(.*)/ # test every branch which starts with "pages-"
+  - add-travis
+env:
+  global:
+  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
+addons:
+  apt:
+    packages:
+    - libcurl4-openssl-dev
+sudo: false # route your build to the container-based infrastructure for a faster build
+cache: bundler
+notifications:
```

Comment:
Mentioned on https://docs.travis-ci.com/user/languages/ruby/  it depends on ruby versions
```
 before_install:
    - gem update --system
    - gem install bundler
```